README.md

```diff
@@ -2,8 +2,6 @@
 Let's collect the data of Woman on GitHub, who inspired you to code more, learn more.
 
 **Note:** Please maintain the below list in alphabetical order.
-
-[alicetragedy](https://github.com/alicetragedy)
```

Comment:
No issues @thefourtheye. :+1: 

I know sometimes it becomes hard to control the words and as I said this was to be "extra" polite so this was harder. Avoiding these things create a friendly and comforting environment for the new contributors and that is all, what we all want. :smile: :star: